resources/handler.yaml

```diff
@@ -39,3 +39,5 @@ spec:
       remoteEndpointIpTagKey: destination.ip
       remoteEndpointNameTagKey: destination.workload.name
       swapLocalRemoteEndpoints: true
+      globalSpanTags:  # any tags added below will be added to all spans
```

Comment:
This is kind of redundant.  You can already do global span tags by just specifying an additional tag in https://github.com/signalfx/signalfx-istio-adapter/blob/master/resources/tracespan.yaml#L25 when you specify the tracespan format.

You should just add this as a value in the Helm chart and make the template that renders the tracespan.yaml add these at the end.  It will be simpler than passing everything through the adapter and having additional logic in the adapter code to add them.

Reply:
Note that for string literal values you have to put them in double quotes inside single quotes in the spanTags section or else it will try to evaluate them as variables.